pipeline/01_parser.py

```
@@ -1,89 +1,533 @@
+                if name.startswith("Contents/section") and name.endswith(".xml")
+            )
+            if not section_files:
+                print(f"  [HWPX] section XML 없음: {os.path.basename(file_path)}")
+                return ""
+
+            p_tag   = "{http://www.hancom.co.kr/hwpml/2012/paragraph}p"
+            tbl_tag = "{http://www.hancom.co.kr/hwpml/2012/paragraph}tbl"
+
+            for sec_name in section_files:
+                with z.open(sec_name) as f:
+                    root = ET.parse(f).getroot()
```

Ruff (0.15.12)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)